doc/modules/language-guide/pages/basic-concepts.adoc

```diff
@@ -2,9 +2,9 @@
 :proglang: Motoko
 :company-id: DFINITY
 
-{proglang} is designed for distributed programming with actors.
+{proglang} is designed for distributed programming with actors. When programming on the Internet Computer, Motoko actors represent  {IC} canister smart contracts with Candid interfaces.
```

Comment:
I was looking for precisely this sentence, to see where and how the terms "actor" and "canister" are first connected in the text.

I wonder if we could expand this statement slightly to help address @osa1's other comment (now resolved):

> When programming on the Internet Compute in Motoko, each **actor** represents an **{IC} canister smart contract** with a Candid interface, whether written in Motoko, Rust, Wasm or some other language that compiles to Wasm.

> Within Motoko, we use the term *actor* to refer to any canister, authored in any language that deploys to the {IC}.

> The role of Motoko is to make these actors easy to author, and easy to use programmatically, once deployed.

I'm offering those statements as suggestions, but not suggested edits; so feel free to use or not.

BTW, I learned from Norman Ramsey's technical writing notes that using singular forms of nouns is more clear in definitions.

Reply:
That's nice, I'll use that. I agree with Norman.

Reply:
```suggestion
{proglang} is designed for distributed programming with actors. 
When programming on the Internet Computer in Motoko, each **actor** represents an **{IC} canister smart contract** with a Candid interface, whether written in Motoko, Rust, Wasm or some other language that compiles to Wasm.
Within Motoko, we use the term *actor* to refer to any canister, authored in any language that deploys to the {IC}.
The role of Motoko is to make these actors easy to author, and easy to use programmatically, once deployed.
```